help/src/lib.rs

```diff
@@ -23,6 +23,7 @@ const COMMANDS: &[Help] = &[
     rad_untrack::HELP,
     rad_sync::HELP,
     rad_ens::HELP,
+    rad_ci::HELP,
```

Comment:
Let's not include it in the list of core commands yet, until it gets more testing.

Reply:
done